src/site/layouts/tests/vamc/fixtures/health_care_local_facility_no_phone_mental.json

```diff
@@ -664,6 +667,131 @@
     },
     {
       "entity": {
+        "status": false,
```

Comment:
This entry needs to be filtered out by the template change, otherwise it will throw an axe error in the unit test.